src/lasp_membership.erl

```diff
@@ -80,19 +80,19 @@ init([]) ->
     {reply, term(), #state{}}.
 
 handle_call(Msg, _From, State) ->
-    _ = lager:warning("Unhandled messages: ~p", [Msg]),
+    _ = lager:warning("Unhandled call: ~p", [Msg]),
     {reply, ok, State}.
```

Comment:
Bit of a nitpick, since this has no call's it isn't going to turn into an issue here, but I've seen this cause bugs to go unnoticed and become very difficult to debug: do a `noreply` here instead of `reply`.